_data-prepper/pipelines/configuration/processors/add-entries.md

```diff
@@ -18,6 +18,10 @@ You can configure the `add_entries` processor with the following options.
 | :--- | :--- | :--- |
 | `entries` | Yes | A list of entries to add to an event. |
 | `key` | Yes | The key of the new entry to be added. Some examples of keys include `my_key`, `myKey`, and `object/sub_Key`. |
+| `metadata_key` | Yes | The key for the new metadata attribute. Argument must be a literal string key and not a JsonPointer. Either one string key or `metadata_key` is required. |
+| `format` | No | A format string to use as the value of the new entry, for example, `${key1}-${key2}` where `key1` and `key2` are existing keys in the event. Required if `value` and `value_expression` is not specified. |
```

Comment:
The last sentence needs some clarification. Do we mean "Required if neither `value` nor `value_expression` is specified" or "Required if both `value` and `value_expression` are not specified"?

Reply:
The first one.